re/api/endpoints.go

```diff
@@ -92,21 +92,21 @@ func listRulesEndpoint(s re.Service) endpoint.Endpoint {
 	}
 }
 
-func upadateRuleStatusEndpoint(s re.Service) endpoint.Endpoint {
+func deleteRuleEndpoint(s re.Service) endpoint.Endpoint {
```

Comment:
You should be able to switch the rule on and off, i.e. to change the status to without deleting it.